src/pyansys_sphinx_theme/__init__.py

```diff
@@ -27,9 +27,7 @@ def setup(app):
     """Connect to the sphinx theme app."""
     theme_path = get_html_theme_path()[0]
     app.add_html_theme("pyansys_sphinx_theme", theme_path)
-    theme_css_path = os.path.join(theme_path, "static", "css", "pyansys_sphinx_theme.css")
-    if not os.path.isfile(theme_css_path):
-        raise FileNotFoundError(f"Unable to locate pyansys-sphinx theme at {theme_css_path}")
```

Comment:
it is because, i think the file is loading in to static/ css when we are doing add_css_file. but we are putting the full path make it to load static/ css / html_path which we are adding

Reply:
You are right, I just checked the official docs: https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.add_css_file